Lab sim refactor

MikeWrock opened the pull request.

MikeWrock commented on src/mobile_manipulation_config/description/scene.xml:
```diff
@@ -48,8 +21,22 @@
     />
+    <site
+      name="scene_camera_optical_frame"
+      pos="0.0 -2.0 3.5"
```
@mkhansenbot I grabbed these numbers from [here](https://github.com/PickNikRobotics/moveit_studio_ur_ws/blob/main/src/mobile_manipulation_config/description/ur5e.xml#L207-L219), the `pos` does not match. The camera angle is way off too so I'm not eve sure if they are aligned. Do you have a different (working) camera frame?

mewtwoshaurd reviewed (changes requested):
A few things:
On my laptop, I'm getting a lot of `Mujoco model timestep not running in realtime` errors -- I know that generally indicates poor performance, which I think deleting the stls and textures should help, so something seems off.
Outside of that, the robot seems in a completely unrecoverable state? I can't get teleoperate to move it anywhere, and it looks strangely positioned compared to what I'm used to with the simulator:
![image](https://github.com/user-attachments/assets/05fed343-becc-4abc-8d81-e6debc2c947e)
I think you mentioned the `pos` variables may be the issue, if you need any help  testing that please let me know!

MikeWrock commented on src/arm_on_rail_sim/config/config.yaml:
```diff
@@ -17,6 +17,7 @@ hardware:
       path: "config/moveit/picknik_ur.srdf"
     urdf_params:
       - mujoco_model: "description/scene.xml"
+      # - mujoco_model: "description/lab_scene.xml"
```
This is how to switch between the lab and simplified table world, I just changed the default to lab environment

MikeWrock commented:
@mewtwoshaurd I added a keyframe so the arm doesn't spawn in a goofy pose, and switched the default to using the lab world. Could you please git it another test?

MikeWrock commented:
Also check out this PR for more context: https://github.com/PickNikRobotics/picknik_accessories/pull/12/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

mewtwoshaurd commented:
works much better, thanks Michael!
